core/lib/Thelia/Controller/Admin/HomeController.php

```diff
@@ -74,61 +86,100 @@ public function loadStatsAjaxAction()
             return $response;
         }
 
-        $data = new \stdClass();
+        $cacheEnable = intval(ConfigQuery::read("admin_cache_home_stats_enable", '1'));
 
-        $data->title = $this->getTranslator()->trans("Stats on %month/%year", array('%month' => $this->getRequest()->query->get('month', date('m')), '%year' => $this->getRequest()->query->get('year', date('Y'))));
+        $cacheContent = false;
 
-        /* sales */
-        $saleSeries = new \stdClass();
-        $saleSeries->color = $this->getRequest()->query->get('sales_color', '#adadad');
-        $saleSeries->data = OrderQuery::getMonthlySaleStats(
-            $this->getRequest()->query->get('month', date('m')),
-            $this->getRequest()->query->get('year', date('Y'))
-        );
+        if ($cacheEnable) {
```

Comment:
variable cacheEnable is used only once, so use directly the ConfigQuery method here.